README.md

```diff
@@ -52,22 +111,21 @@ Since tokens are private, we suggest adding them as [GitHub secrets](https://hel
 
 ### Skipping branches
 
-Since GitHub actions take string inputs, you must generate a regex which will work for all sets of branches you want to ignore. This is useful for merging protected/default branches into other branches. Check out some [examples in the tests](https://github.com/ClearTax/pr-lint/blob/2bb72327ef04ab028caf84a099ffbc08b4dd0959/__tests__/utils.test.ts#L30-L41).
+Since GitHub actions take string inputs, `skip-branches` must be a regex which will work for all sets of branches you want to ignore. This is useful for merging protected/default branches into other branches. Check out some [examples in the tests](https://github.com/cleartax/pivotal-lint/blob/2bb72327ef04ab028caf84a099ffbc08b4dd0959/__tests__/utils.test.ts#L30-L41).
 
-`pr-lint` already skips PRs which are filed by bots (eg dependabot). You can add more bots to [this list](https://github.com/ClearTax/pr-lint/blob/2bb72327ef04ab028caf84a099ffbc08b4dd0959/src/constants.ts#L4-L6).
+`pivotal-lint` already skips PRs which are filed by bots (for eg. [dependabot](https://github.com/marketplace/dependabot-preview)). You can add more bots to [this list](https://github.com/cleartax/pivotal-lint/blob/2bb72327ef04ab028caf84a099ffbc08b4dd0959/src/constants.ts#L4-L6), or add the branch-format followed by the bot PRs to the `skip-branches` option.
 
-### Versions
+### Semantic Versions
 
-If you want more stability in versions than `@master` you can also use the [semantic releases for pr-lint](https://github.com/ClearTax/pr-lint/releases).
+If you want more stability in versions of `pivotal-lint` than `@master` you can also use the [semantic releases for pivotal-lint](https://github.com/cleartax/pivotal-lint/releases).
 
 Example:
 
 ```yaml
 # ...
-  pr_lint:
     steps:
-    - name: Verify Pivotal story ID & add relevant labels
-      uses: cleartax/pr-lint@v2.1.0
+    - uses: cleartax/pivotal-lint@v2.1.0
```

Comment:
Please add `pivotal-lint@latest`

Reply:
Sounds good! Done for the `@master` reference instead of this one!